packages/plugin-devtools/test/devtools.test.ts

```diff
@@ -0,0 +1,735 @@
+            await longNextTick();
+            expect(line.classList.contains('selected')).to.equal(true);
+
+            const about = devtools.querySelector('devtools-info .about');
+            const aResult = '<div class="about"><span class="type">Command</span> setRange</div>';
+            expect(about.outerHTML).to.equal(aResult);
+
+            const properties = devtools.querySelector('devtools-info .properties table');
+            const pResult =
+                '<table>' +
+                '<tbody>' +
+                '<tr><td>vRange</td><td><table><tbody>' +
+                '<tr><td> direction </td><td>FORWARD</td></tr>' +
+                '<tr><td> start </td><td> 12 (b) </td></tr>' +
+                '<tr><td> end </td><td> 12 (b) </td></tr>' +
+                '</tbody></table></td></tr>' +
+                '</tbody>' +
+                '</table>';
+            expect(properties.outerHTML).to.equal(pResult);
+        });
+        it('should select "insertText" with arrow', async () => {
+            const subpanel = devtools.querySelector('devtools-panel.active mainpane-contents');
+            const line = subpanel.querySelector('.selectable-line');
+            line.nextElementSibling.dispatchEvent(
+                new KeyboardEvent('keydown', {
+                    key: 'ArrowUp',
+                    code: 'ArrowUp',
+                    bubbles: true,
+                }),
+            );
+            await longNextTick();
+            expect(line.classList.contains('selected')).to.equal(true);
+
+            const about = devtools.querySelector('devtools-info .about');
+            const aResult = '<div class="about"><span class="type">Command</span> insertText</div>';
+            expect(about.outerHTML).to.equal(aResult);
+
+            const properties = devtools.querySelector('devtools-info .properties table');
+            const pResult =
+                '<table>' +
+                '<tbody>' +
+                '<tr><td>value</td><td>z</td></tr>' +
+                '<tr><td>elements</td><td>[object Set]</td></tr>' +
+                '<tr><td>origin</td><td>EventNormalizer</td></tr>' +
+                '</tbody>' +
+                '</table>';
+            expect(properties.outerHTML).to.equal(pResult);
+        });
+        it('should select "setRange" with arrow', async () => {
+            const selected = devtools.querySelector(
+                'devtools-panel.active mainpane-contents .selectable-line.selected',
+            );
+            selected.dispatchEvent(
+                new KeyboardEvent('keydown', {
+                    key: 'ArrowDown',
+                    code: 'ArrowDown',
+                    bubbles: true,
+                }),
+            );
+            await longNextTick();
+            expect(selected.nextElementSibling.classList.contains('selected')).to.equal(true);
+        });
+        it('should not change the selection with other key', async () => {
```

Comment:
Isn't there already a test that tests this ?

Reply:
No, here we test that it doesn't matter (=> 100% coverage)